src/apps/lwaftr/fragmentv4.lua

```diff
@@ -0,0 +1,100 @@
+module(..., package.seeall)
+
+local constants = require("apps.lwaftr.constants")
+local packet = require("core.packet")
+local ffi = require("ffi")
+
+local band, bor, lshift = bit.band, bit.bor, bit.lshift
+local ntohs, htons = ffi.C.ntohs, ffi.C.htons
+
```

Comment:
It's a common LuaJIT idiom to cache the C namespace into a local:
```lua
local C = ffi.C
```
However, LuaJIT docs doesn't recommend to cache C functions into locals. The explanation is in this article http://luajit.org/ext_ffi_tutorial.html : 

"The situation is a bit different with C function calls via the FFI library. The JIT compiler has special logic to eliminate all of the lookup overhead for functions resolved from a C library namespace! Thus it's not helpful and actually counter-productive to cache individual C functions"